sources/web/datalab/static/datalab.js

```diff
@@ -102,7 +105,9 @@ define([
   'static/notebook-app',
   'static/notebook-list',
   'static/websocket'
-], function(ipy, events, dialog, utils, security, appbar, editapp,
+], function(ipy,     events, dialog, utils, security, appbar, editapp,
             minitoolbar, notebookapp, notebooklist, websocket) {
-  initializeDataLab(IPython, events, dialog, utils, security);
+     initializeDataLab(
+            IPython, events, dialog, utils, security, appbar, editapp,
```

Comment:
Sounds good to me. Done. I didn't know why IPython was used instead of ipy, so I didn't change it earlier.